### PR TITLE
Ci/create GitHub actions workflow

### PR DESCRIPTION
**Pull request by PProfizi**

### CODE REVIEW

**Comment by PProfizi** [comment]
@MaxJPRey I've added coverage reporting to my PR for switching to GitHub Actions for PyDPF-Post. Is there everything needed?

**Comment by PProfizi** [comment]
Once the PR is validated for merging I'll have to change the settings of the repo, which right now requires pyansys.pydpf-post (Windows) to have run, which will be obsolete and replaced by the equivalent GitHub workflow.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ env.PACKAGE_NAME }}_doctest
+          path: junit/test-doctests-results.xml
+        if: always()
+
+      - name: Test Post API
+        run: |
+          cd tests
+          pytest --cov=ansys.dpf.post --cov-report=xml --cov-report=html --log-level=ERROR --junitxml=junit/test-results.xml --reruns 2 .
```
Comment (MaxJPRey) [inline .github/workflows/ci.yml:139]: @PProfizi You can add the verbose option if you want to get detailed info when test failure occurs.
```suggestion
          pytest -v --cov=ansys.dpf.post --cov-report=xml --cov-report=html --log-level=ERROR --junitxml=junit/test-results.xml --reruns 2 .
```

**Review by MaxJPRey — APPROVED** [review]

File: ansys/dpf/post/dpf_solution.py [hunk 2]
```diff
@@ -51,17 +51,35 @@ def time_freq_support(self):
...
         --------
         >>> from ansys.dpf import post
-        >>> result = post.result("file.rst")
-        >>> print(result.get_result_info())
+        >>> from ansys.dpf.post import examples
+        >>> solution = post.load_solution(examples.static_rst)
+        >>> print(solution.get_result_info()) # doctest: +NORMALIZE_WHITESPACE
+        Static analysis
+        Unit system: MKS: m, kg, N, s, V, A, degC
+        Physics Type: Mecanic
```
[the remark this answers is not in the document]
Comment (PProfizi) [reply ansys/dpf/post/dpf_solution.py:64]: @MaxJPRey Yes this is a known typo on the server side.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (PProfizi) [reply .github/workflows/ci.yml:139]: Will do, thx!

**Pull request merged**